tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -0,0 +1,104 @@
+    url: ['citation_public_url', 'og:url'],
+};
+
+function getMetaTagsFor(assert: Assert, prop: string): Array<string|null> {
+    const terms = tagsMap[prop];
+
+    const metaTags = terms.map(term => {
+        const k = term.includes('og:') ? 'property' : 'name';
+        const selector = `meta[${k}="${term}"]`;
+        const elements = [...document.querySelectorAll(selector)];
+        return elements.map((el: Element) => el.getAttribute('content'));
+    });
+
+    if (metaTags.length > 1) {
+        assert.ok(metaTags.every(m => JSON.stringify(m) === JSON.stringify(metaTags[0])));
+    }
+
+    return metaTags[0];
+}
+
+type RegistrationAttrs = 'title'|'description'|'dateModified'|'dateRegistered';
+
+module('Registries | Acceptance | overview.overview', hooks => {
+    setupEngineApplicationTest(hooks, 'registries');
+    setupMirage(hooks);
+
+    hooks.beforeEach(function(this: TestContext) {
+        this.store = this.owner.lookup('service:store');
+    });
+
+    test('Check head meta tags', async function(this: TestContext, assert: Assert) {
+        const reg = server.create('registration', {
+            description: 'Sun',
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+            affiliatedInstitutions: server.createList('institution', 2, {}),
+            tags: ['Social Media', 'Education'],
+            currentUserPermissions: ['admin'],
+        }, 'withContributors');
+
+        const registration: RegistrationModel = await this.store.findRecord('registration', reg.id, {
+            include: ['registration_schema', 'institution'],
+        });
+
+        await visit(`/${registration.id}/`);
+
+        let metaTag;
+
+        // title, description
+        ['title', 'description'].forEach((t: RegistrationAttrs) => {
+            [metaTag] = getMetaTagsFor(assert, t);
+            assert.equal(metaTag, registration[t],
+                `Registration.${t} does not match ${tagsMap[t]} head meta`);
+        });
+
+        // dateRegistered, datemodified
+        ['dateModified', 'dateRegistered'].forEach((t: RegistrationAttrs) => {
+            [metaTag] = getMetaTagsFor(assert, t);
+            assert.equal(metaTag, moment(registration[t]).format('YYYY-MM-DD'),
+                `registration.${t} does not match ${tagsMap[t]} head meta`);
+        });
+
+        // registration url
+        [metaTag] = getMetaTagsFor(assert, 'url');
+        assert.equal(metaTag, pathJoin(config.OSF.url, registration.id),
+            `registration url does not match ${tagsMap.url} head meta`);
+
+        // registrations tags
+        metaTag = getMetaTagsFor(assert, 'tags');
+        assert.ok(JSON.stringify(metaTag) === JSON.stringify(registration.tags),
+            `registration.tags do not match ${tagsMap.tags} head meta`);
+
+        const contributors = await registration.loadAll('contributors');
+        const institutions = await registration.loadAll('affiliatedInstitutions');
```

Comment:
Instead of using `loadAll` in a test, how about using mirage data directly?
```ts
const contributors = reg.contributors.models;
const institutions = reg.affiliatedInstitutions.models;
```